.circleci/config.yml

```diff
@@ -50,10 +50,6 @@ workflows:
             - sb/normalize
       - sb/publish-artifacts:
           context: project-bot
-          filters:
```

Comment:
why is this removed? we still want to deploy branches to tools-dev, right?

Reply:
without the filter and https://github.com/cambiahealth/tools-env-configs/pull/67/files i thought we would be able to deploy branches to tools-dev? Doesn't the removal of the filter all branchs builds to be published to artifactory?

Reply:
@figadore - reese am i missing something with how filters work? I'd like to get this merged today to get this service into the next round of scans over the weekend.